dev/test/specs/starter-test.js

```diff
@@ -199,30 +237,9 @@ function clickInParticles(slotName, selectors, textQuery) {
 
 describe('test basic arcs functionality', function() {
```

Comment:
Now this description doesn't seem right.. maybe 'test Arcs demo flows'?

Reply:
(low priority, though).

Reply:
done